scripts/environment/bootstrap-ubuntu-20.04.sh

```diff
@@ -206,3 +206,5 @@ EOF
 rustc-wrapper = "$RUSTC_WRAPPER"
 EOF
 fi
+
+sudo chmod 755 /var/lib/vector
```

Comment:
I think the `sudo` is unnecessary here. We already do things like `apt-get` above without `sudo`.